slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -123,6 +123,10 @@ void SLAMServiceImpl::DetermineActionMode() {
             return;
         }
     }
+    if (map_rate_sec.count() == 0) {
+        throw std::runtime_error(
+            "In localization mode but couldn't find a map to localize on");
```

Comment:
[opt] 
`map_rate_sec set to localization mode but couldn't find an apriori map`
or
`set to localization mode (map_rate_sec = 0) but couldn't find apriori map to localize on`